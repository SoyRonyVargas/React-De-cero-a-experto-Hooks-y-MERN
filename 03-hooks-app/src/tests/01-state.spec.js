import CounterApp from '../components/01-useState/counterApp'
import { shallow } from 'enzyme'
import React from 'react'

describe('Montando el proyecto', () => { 

    it("montando el componente" , () => {

        const wrapper = shallow(<CounterApp/>)

        expect(wrapper).toMatchSnapshot()

    })
    
})