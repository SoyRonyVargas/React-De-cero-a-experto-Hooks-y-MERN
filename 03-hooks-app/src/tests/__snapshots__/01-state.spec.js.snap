// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Montando el proyecto montando el componente 1`] = `
<Fragment>
  <h1>
    Counter1 
    1
  </h1>
  <h1>
    Counter2 
    5
  </h1>
  <hr />
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    +1
  </button>
</Fragment>
`;
